test/libsolidity/smtCheckerTests/loops/while_bmc_iterations_semantics_3.sol

```diff
@@ -7,14 +7,15 @@ contract C {
 				y = 1;
 			++x;
 		}
-        // BMC loop iteration setting is more than enough to leave the loop
+		// BMC loop iteration setting is more than enough to leave the loop
 		assert(x == 3); // should hold
-		assert(y == 0); // should hold
+		assert(y == 1); // should fail
```

Comment:
Oh, you are changing this one to fail. Why switching the expectations in the `while_bmc_iterations_semantics` `2` and `3` and not keeping the original one?

Reply:
Sorry Martin. I messed up a bit here. I think I wanted them to be consistent and changed in both places.
Let's agree on `should fail` version.

Reply:
I made those tests consistent. Let me know how you like them now.